sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -399,7 +399,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
       try {
         val plan = sparkSession.sessionState.executePlan(viewDef)
         sparkSession.sharedState.cacheManager.uncacheQuery(
-          sparkSession, plan.analyzed, cascade = false)
+          sparkSession, plan.analyzed, cascade = true)
```

Comment:
shall we check if it's view with analyzed plan or view with SQL text?

Reply:
Oops forgot to update here. Will update and add test cases.